test/DynamoCoreWpfTests/PresetViewModelTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Windows;
+
+using SystemTestServices;
+
+using Dynamo;
+using Dynamo.Controls;
+using Dynamo.Models;
+using Dynamo.Nodes;
+using Dynamo.Selection;
+using Dynamo.Services;
+using Dynamo.Tests;
+using Dynamo.ViewModels;
+using NUnit.Framework;
+
+namespace DynamoCoreWpfTests
+{
+    [TestFixture]
+    public class PresetViewModelTests : DynamoViewModelUnitTest
+    {
```

Comment:
Yes. One of the other task had the same file with a different test. That particular task requires DyanmoViewmodel to test.